post.hbs

```diff
@@ -31,8 +21,28 @@
                             </span>
                         {{/foreach}}
                     </span>
+                    <span>
+                        Reading Time:&nbsp;
+                        {{reading_time}}
+                    </span>
                 </span>
-
+                
+                <span class="flex flex-auto content-center gap-2 justify-center"> 
```

Comment:
Could we use the same class `gh-post-meta`? The tags list should look identical to the other metadata (with everything capitalised by CSS).

Reply:
Similarly, for the below tags, feel free to use `gh-post-meta-content-item`. We can migrate away from these classes in a separate PR.